content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+the rules defined by policies.
+
+A policy defines one or more criteria that your artifacts should fulfill. For
+example, one of the default policies in Docker Scout is the **Critical
+vulnerabilities** policy, which proclaims that your artifacts must not contain
+any critical vulnerabilities. If an artifact contains one or more
+vulnerabilities with a critical severity, that artifact will have failed the
+policy check.
+
+In Docker Scout, policies are not meant to be used for gate-keeping. The
+purpose of policies is to help you ratchet forward your security and supply
+chain stature. For artifacts that fail the policy check, you can track how the
+fail gap changes over time. This lets you see if your artifact is improving or
+deteriorating relative to policy.
+
+Policies don't necessarily have to be related to application security and
+vulnerabilities. You can use policies to measure and track other aspects of
+supply chain management as well, such as base image dependencies and
+open-source licenses.
+
+## Default policies
+
+Docker Scout ships the following three out of the box policies:
+
+- [Critical and high vulnerabilities with fixes](#critical-and-high-vulnerabilities-with-fixes)
+- [Critical vulnerabilities](#critical-vulnerabilities)
+- [Packages with GPL3+ licenses](#packages-with-gpl3-licenses)
+
+These policies are turned on for your active repositories by default. There's
+currently no way to turn off or configure these policies.
+
+### Critical and high vulnerabilities with fixes
+
+This policy requires that your artifacts aren't exposed to known
+vulnerabilities with a critical or high severity, and where there's a fix
+version available. Essentially, this means that there's an easy fix that you
+can deploy for images that fail this policy: upgrade the vulnerable package to
+a version containing a fix for the vulnerability.
+
+This policy is unfulfilled if an artifact is affected by one or more
+vulnerability where a fix version is available.
+
+### Critical vulnerabilities
+
+This policy requires that your artifacts contain no known critical
+vulnerabilities. The policy is unfulfilled if your artifact contains one or
+more critical vulnerabilities.
+
+This policy flags all critical vulnerabilities, whether or not there's a fix
+version available.
+
+### Packages with GPL3+ licenses
+
+This policy requires that your artifacts don't contain packages distributed
+under a GPL3+ [copyleft](https://en.wikipedia.org/wiki/Copyleft) license.
+
+This policy is unfulfilled if your artifacts contain one or more packages with
+a violating license.
+
+## View policy status
+
+The **Overview** tab of the [Docker Scout Dashboard](https://scout.docker.com/)
+displays a summary of recent changes in policy for your repositories.
+This summary shows images that have seen the most change in their policy
+evaluation between the most recently analyzed tag and the previous tag.
```

Comment:
Technically this looks at most recent and previous *image*, regardless of whether they are tagged.

Reply:
I like the explanation here! Short, sweet and to the point 😃